data/human/deep missions.txt

```diff
@@ -3365,19 +3361,20 @@ event "mani refit"
 
 
 mission "Mani Refit"
-	name "Mani Refit"
+	name "Mani Refitted"
 	to offer
 		has "event: mani refit"
-		has "Home for Skadenga 5: done"
 	description "Head back to <destination> and meet up with the Transport Union's Fleet."
 	source
-		government "Republic"
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not planet "Mani"
 	destination "Mani"
 	on offer
-		dialog
```

Comment:
I strongly disagree with the change here. As a dialog it looks and _feels_ like a message.

Reply:
But is the intention that you're able to decline? As stated by Anarchist, *all* dialogs give the option to accept or decline, whereas conversations are up to the content creator as to how they end. If we don't want the option to decline then we need to use a conversation regardless of how we feel about the aesthetics of a dialog box.

Reply:
Yeah, let them decline. It's their pilot.